tests/waterservices_test.py

```diff
@@ -213,12 +241,16 @@ def test_get_qwdata(requests_mock):
 def test_get_qwdata_site_value_types(requests_mock, site_input_type_list):
     """Tests get_qwdata method for valid input types for the 'sites' parameter"""
     _format = "rdb"
-    site = '01491000'
-    request_url = 'https://nwis.waterdata.usgs.gov/nwis/qwdata?site_no={}' \
-                  '&qw_sample_wide=qw_sample_wide&agency_cd=USGS&format={}&pm_cd_compare=Greater+than' \
-                  '&inventory_output=0&rdb_inventory_output=file&TZoutput=0&rdb_qw_attributes=expanded' \
-                  '&date_format=YYYY-MM-DD&rdb_compression=value&submitted_form=brief_list'.format(site, _format)
-    response_file_path = 'data/waterdata_qwdata.txt'
+    site = "01491000"
+    request_url = (
+        "https://nwis.waterdata.usgs.gov/nwis/qwdata?site_no={}"
+        "&qw_sample_wide=qw_sample_wide&agency_cd=USGS&format={}&pm_cd_compare=Greater+than"
+        "&inventory_output=0&rdb_inventory_output=file&TZoutput=0&rdb_qw_attributes=expanded"
+        "&date_format=YYYY-MM-DD&rdb_compression=value&submitted_form=brief_list".format(
+            site, _format
+        )
```

Comment:
This one weirds me out since the `format()` trailing parenthesis is on its own line, and you have to go searching for its pair a couple lines up on a fairly long line. I am not sure about alternatives besides having the entire contents of `format()` on one line, but that messes up another rule in the formatting.